lib/shopify_api/session.rb

```diff
@@ -6,10 +6,9 @@ class ValidationException < StandardError
   end
 
   class Session
-    cattr_accessor :api_key
-    cattr_accessor :secret
-    cattr_accessor :protocol
+    cattr_accessor :api_key, :secret, :protocol, :myshopify_domain, :port
     self.protocol = 'https'
+    self.myshopify_domain = 'myshopify.com'
```

Comment:
I guess we can override this if we are using the api client against a local shopify. I added something similar to shopify_app recently and I think we could just patch this in in actual apps for development.